CMakeLists.txt

```diff
@@ -44,6 +44,7 @@ option(TRITON_ENABLE_PERF_ANALYZER_TS "Enable TorchServe support for Performance
 option(TRITON_ENABLE_EXAMPLES "Include examples in build" OFF)
 option(TRITON_ENABLE_TESTS "Include tests in build" OFF)
 option(TRITON_ENABLE_GPU "Enable GPU support in libraries" OFF)
+option(TRITON_INCLUDE_ZLIB_IN_BUILD "Include libz library in build" ON)
```

Comment:
question: in the pull request comments you mention a "STATIC_LINK_ZLIB" type flag instead of "INCLUDE_ZLIB" type flag - just curious it seems from the description the flag would be to static link or dynamic link - can also discuss offline.

Reply:
copy paste typo `TRITON_INCLUDE_ZLIB_IN_BUILD`